### PR TITLE
add .editorconfig

### PR DESCRIPTION
**Pull request by rpitonak**
This is the first proposal on `.editorconfig`.

I would like to hear your suggestions so we can agree here on the final version.

### CODE REVIEW

**Comment by dhodovsk** [comment]
LGTM

**Review by TomasTomecek — APPROVED** [review]
LGTM

**Review by lachmanfrantisek — APPROVED** [review]

**Comment by TomasTomecek** [comment]
@phracek and @jscotka remain :D

**Comment by phracek** [comment]
LGTM

**Comment by eliskasl** [comment]
LGTM :)

**Review by jpopelka — APPROVED** [review]

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+end_of_line = lf
+charset = utf-8
+
+[*.py]
+max_line_length = 100
+
+[Makefile]
+indent_style = tab
+
+[*.{json,y{a,}ml,sh}]
```
Comment (jscotka) [inline .editorconfig:19]: for ``*.sh`` we prefer to use 4 spaces from my perspective. But I can live with 2 :-)

**Review by jscotka — COMMENTED** [review]
just sh extension, but really it is not so important

File: .editorconfig (hunk 1 above)
Comment (TomasTomecek) [reply .editorconfig:19]: I'm using two as well: https://github.com/TomasTomecek/dotfiles/blob/master/vim/.vimrc#L135

**Comment by TomasTomecek** [comment]
\o/

I'm gonna create a card to write a bot which will sync files from kwaciaren in all our repos

**Pull request merged**